### DISCUSSION TITLE
Pull Request for Core V1

### DISCUSSION THREAD

**Discussion by nigelb** [sparkfun/Arduino_Apollo3#436] [Q&A] [answered]
Hi,

According to the [Development Lifecycle](https://github.com/sparkfun/Arduino_Apollo3/blob/main/docs/CONTRIBUTING.md#development-lifecycle)  section of the contribution guide, pull requests are to be made to the `dev` branch.

Is this only for Core V2?
To create a pull request for Core V1, does one just issue the pull request to the [v1](https://github.com/sparkfun/Arduino_Apollo3/tree/v1) branch?

**Comment by Wenn0101** [comment, marked as answer]
Yea this is only for the v2 core. Since there is less activity and less of a build process to make the v1 core, you can just make pull requests directly to the v1 branch.
